src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -132,26 +134,21 @@ const ContactDetails = ({
               </ErrorAlert>
             )}
             <p className="line-height-body-5">
-              The EASi System Intake process can guide you through all stages of
-              your project, connecting you with the resources, people and
-              services that you need. Please complete and submit this CMS IT
-              Intake form to engage with the CMS IT Governance review process.
-              This is the first step to receive a CMS IT LifeCycle ID. Upon
-              submission, you will receive an email promptly from the
-              IT_Governance mailbox, and an IT Governance Team member will reach
-              out regarding next steps.
+              {t('contactDetailsForm.intakeProcessDescription')}
             </p>
 
             <div className="tablet:grid-col-6 margin-bottom-7">
               <MandatoryFieldsAlert />
-              <PageHeading>Contact details</PageHeading>
+              <PageHeading>{t('contactDetailsForm.heading')}</PageHeading>
               <Form>
                 {/* Requester Name */}
                 <FieldGroup
                   scrollElement="requester.name"
                   error={!!flatErrors['requester.name']}
                 >
-                  <Label htmlFor="IntakeForm-Requester">Requester</Label>
+                  <Label htmlFor="IntakeForm-Requester">
+                    {t('fields.requester')}
```

Comment:
You can remove this. This should be in the new stuff you've written. Just make sure all the use cases of `fields.requester` are covered before deleting.

Reply:
I would convert all other instances to use contactDetailsForm.requester? The other places this is used is in the grt summary and request repo.

It feels weird to use contactDetailsForm in those other places, I like the fields section being for general fields that are used across the application.

Reply:
Possibly, it feels less weird to be because the intake is the source of truth for those fields on those respective pages.

Whereas previously, with `csvHeadings`, it was getting used in the intake form; felt backwards.

As for this, I'm okay with what you decide. It's something we can re-visit later, if needed.

Reply:
that is true, i think if we're going to share it across various places i might change contactDetailsForm to just contactDetails. I'll play around with it and see what I can come up with. thanks!